File: Questao1/RPC/Cliente.py
import xmlrpc.client

nome = input("Qual o nome do funcionário? ")
cargo = input("Qual o cargo do funcionário? ")
salario = float(input("Qual o salário do funcionário? "))

with xmlrpc.client.ServerProxy("http://localhost:5000/") as proxy:
    print("O novo salário de", nome, "é:", "%s" %
          str(proxy.salarioReajustado(cargo, salario)))

File: Entrega_da_Aula_20-12-2022/Questao1/RPC/Cliente.rb
require "xmlrpc/client"

server = XMLRPC::Client.new("localhost", "/RPC2", 5000)
begin
    print "Qual o nome do funcionário? "
    nome = gets
    print "Qual o cargo do funcionário? "
    cargo = gets.chomp
    print "Qual o salário do funcionário? "
    salario = gets
    result = server.call("salarioReajustado", cargo, salario.to_f)
    puts result
rescue XMLRPC::FaultException => e
    puts "Error:"
    puts e.faultCode
    puts e.faultString
end